cypress/integration/login.js

```diff
@@ -22,4 +22,11 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 15')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+    it('Check if date is selected as today', function () {
+        const todaysDate = Cypress.moment().format('DD')
+        cy.get('.calendar--today').should('contain', todaysDate)
+        cy.get('.calendar--selected').should('be.visible')
```

Comment:
If I would need to assume that this line should fulfill what the name of this test case implies: "'Check if the date is selected as today'" I could not be for sure, because cy.get('.calendar--selected') would be visible if another day is selected, there is no asserting done that today's date is selected. 

You might use something like cy.get('.calendar--selected').should('have.class', 'calendar--today')